### PR TITLE
Send value for a gender when creating a patient

### PR DESCRIPTION
**Pull request by Johngtka**
- [x] fixed sending gender value

### CODE REVIEW

File: src/app/user-input-dialog/user-input-dial.component.ts [hunk 1]
```diff
@@ -56,5 +56,6 @@ export class UserInputDialogComponent {
                     SNACK_TYPE.error,
                 ),
         });
+        // console.log(patient);
```
Comment (macrog) [inline src/app/user-input-dialog/user-input-dial.component.ts:59]: not needed, please remove

File: src/app/user-input-dialog/user-input-dial.component.html [hunk 2]
```diff
@@ -6,87 +6,68 @@ <h2 mat-dialog-title>{{ 'PATIENT_CREATE.DIALOG.TITLE' | translate }}</h2>
...
-            (click)="addPatient()"
-            cdkFocusInitial
-        >
+        <button mat-raised-button color="primary" [disabled]="!registerForm.valid" mat-button (click)="addPatient()"
+            cdkFocusInitial>
             {{ 'PATIENT_CREATE.DIALOG.REGISTER' | translate }}
         </button>
     </form>
-</mat-dialog-content>
+</mat-dialog-content>
```
Comment (macrog) [inline src/app/user-input-dialog/user-input-dial.component.html:73]: missing empty line at the end of file

**Review by macrog — CHANGES_REQUESTED** [review]
just some little things

**Comment by Johngtka** [comment]
**squash and merge commit message:**
`fix: Form repair(#19 ):`
`- repaired gender send`

**Review by macrog — APPROVED** [review]
👍

**Pull request merged**